staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -285,6 +327,26 @@ func (cfgCtlr *configController) syncOne() bool {
 	return false
 }
 
+func (cfgCtlr *configController) isNews(newPLs []*fctypesv1a1.PriorityLevelConfiguration, newFSs []*fctypesv1a1.FlowSchema) bool {
+	if len(newPLs) != len(cfgCtlr.priorityLevelGenerations) {
+		return true
+	}
+	for _, pl := range newPLs {
+		if pl.Generation != cfgCtlr.priorityLevelGenerations[pl.UID] {
```

Comment:
can a PL be in `newPLs` but not in `priorityLevelGenerations` such that their lengths are same? if so, we should check for the presence of the UID key in the map first

not sure if such a scenario should be considered as news tbh

Reply:
Added comment, hopefully explaining why the code is correct.